### ISSUE TITLE
Module pattern?

### ISSUE THREAD

**Issue by lukasvan3l** [xavcz/papa-clicker#5]
How did the module pattern work out for you?

If I look at these files they can be merged and a lot of code removed. That makes it a lot more readable in my opinion.

https://github.com/xavcz/papa-clicker/blob/master/lib/methods/users.js
https://github.com/xavcz/papa-clicker/blob/master/lib/modules/users.js

**Comment by xavxyz** [comment]
Yep, totally! The way it's currently built makes it hard to read, we'll refactor this!
